pkg/terraform/templates.go

```diff
@@ -0,0 +1,17 @@
+package terraform
+
+const (
+	// BackendTemplate is a template for a terraform backend
+	// TODO(NL): Dynamically write in region here
+	BackendTemplate = `
+terraform {
+	backend "s3" {
+		bucket               = "cnqr-scipian-backend"
+		key                  = "%s"
+		region               = "%s"
+		dynamodb_table       = "cnqr-scipian-backend-locking"
```

Comment:
Wondering if there's a way to pass these in when the CRD is created? That way this isn't hard coded and can be dynamic.

Reply:
See comment above